### PR TITLE
Fe listing listings

### PR DESCRIPTION
**Pull request by ehteht487**

### CODE REVIEW

**Comment by Christonn93** [comment]
@ehteht487
You need to update your work branch from your team branch, if the team branch is not updated you can request a merge from main.


This should resolve the conflict you are having. We can't approve the request before the conflict have been resolves.

**Comment by Jenfer93** [comment]
The FE-Listing should be completely up to date now! If you can update your working branch then it should be withouy conflicts!

File: src/js/api/posts/getListOfListings.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Author: Emilie Herrera Thomsen
+// The Dummy json needs to be replaced with actual endpoint.
+
+import { authBaseFetchOpen } from "../apiBaseFetch.js";
+
+export async function getListOfListings() {
+    const getListingsUrl = "https://dummyjson.com/products";
+
+    const response = await authBaseFetchOpen(getListingsUrl);
```
Comment (akeek) [inline src/js/api/posts/getListOfListings.js:9]: Im not sure if I'm writing the test the correct way, but I get a response that says "fetch is not defined in authBaseFetchOpen"

**Review by akeek — COMMENTED** [review]
Great work! But see comment about the getListOfListings function. I will add the test I made to the QA repo, so you can have a look at that as well. But I think @Jenfer93 wants you to merge anyways :)

**Pull request merged**